src/h5web/providers/mock/utils.ts

```diff
@@ -34,14 +47,13 @@ export function findMockEntity(root: Metadata, path: string): Entity {
 }
 
 export function getMockDataArray(path: string): ndarray {
-  assertAbsolutePath(path);
-
   const dataset = findMockEntity(mockMetadata, path);
-  assertDataset(dataset);
+  assertMockDataset(dataset);
+
   assertNumericType(dataset);
   assertSimpleShape(dataset);
 
-  const value = mockValues[dataset.id as keyof typeof mockValues];
+  const { value } = dataset as MockDataset;
```

Comment:
You could `assertMockDataset` here instead of casting.

Reply:
If I do, I loose the type narrowing of `assertNumericType` and `assertSimpleShape` and can't access `dataset.shape.dims`. 😢

Reply:
:crying_cat_face:

Reply:
Although I could store the dims in a constant before `assertMockDataset`!

Reply:
Or better, the `value`.